keyboards/keebio/nyquist/rev5/config.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2024 Danny Nguyen (danny@keeb.io)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define SPLIT_HAND_PIN GP23
+
+#define USB_VBUS_PIN GP18
+
+#define SERIAL_USART_FULL_DUPLEX
+#define SERIAL_USART_TX_PIN GP8
+#define SERIAL_USART_RX_PIN GP9
+#define SERIAL_USART_PIN_SWAP
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 1000U
+
+#define I2C_DRIVER I2CD0
```

Comment:
is the i2c config necessary?

Reply:
Not necessary, but it's there as a reference if someone wants to use I2C, as breakout pins have been made available on the PCB.